dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentFactoryImpl.java

```diff
@@ -872,6 +873,21 @@ protected Contentlet findContentletByIdentifier(String identifier, Boolean live,
 		}
 	}
 
+	@Override
+    public Contentlet findContentletByIdentifierAnyLanguage(String identifier) throws DotDataException, DotSecurityException {
+        Contentlet contentlet = null;
+        final List<HashMap<String, String>> inodes = new DotConnect()
+                .setSQL("Select inode from contentlet where identifier=?")
```

Comment:
**Question:** I see the old query used to do a `+identifier:" + parent + " +working:true` in the _RulePermissionableUtil_ meaning, was looking for a working version, with the query above you will get any contentlet for that identifier, that includes any old version and even archived ones, that's ok?.
If thats ok I think the documentation should explain what this method would return and if it is not ok maybe we need to query the _version_info_ table directly and ask for a working version?